### PR TITLE
Add response to output

### PR DESCRIPTION
**Pull request by sqren**
Improve logging by adding response object to log output.
Renamed `requestService` to `connectionService`, since it now contains a list of request/response objects, that I call "connections".

@Tradeshift/collaboration

### CODE REVIEW

**Comment by sqren** [comment]
Feedback addressed.

File: src/server/listenerService.js [hunk 1]
```diff
@@ -3,28 +3,28 @@ const http = require('http');
...
 
-listenerService.add = function (port) {
+listenerService.addListener = function (port) {
 	if (listeners[port]) {
 		throw new Error(`Listener already exists: ${port}`);
 	}
 
 	const app = express();
 	app.use(bodyParser.json());
 	app.use((req, res, next) => {
```
Comment (vlezhnin) [inline src/server/listenerService.js]: can this block be moved to listener constructor?

Comment (sqren) [inline src/server/listenerService.js]: Good idea! Looks much nicer now.

**Comment by andrerigon** [comment]
📟

**Pull request merged**